my-app/src/DataStore.jsx

```diff
@@ -10,10 +10,13 @@ import firebaseConfig from './ConstData/firebase_config.js';
 import firebase from 'firebase';
 import 'firebase/firestore';
 
+if (!ENABLE_LOGIN) {
    firebase.initializeApp(firebaseConfig);
    const settings = {timestampsInSnapshots: true};
    firebase.firestore().settings(settings);
    firebase.firestore().enablePersistence();
+}
```

Comment:
Why does the firebase initialization depend on login being enabled?

Reply:
Because in firebase.js, I was initializing the firebase.initializeApp, this is also initializing the app, we only need the app initialized one time.